iced_examples/bottom_panel/src/main.rs

```diff
@@ -0,0 +1,65 @@
+mod applications;
+use applications::{all_apps, App};
+use iced::widget::row;
+use iced::{Command, Element, Theme};
+
+use iced_layershell::reexport::{Anchor, Layer};
+use iced_layershell::settings::{LayerShellSettings, Settings};
+use iced_layershell::Application;
+
+fn main() -> Result<(), iced_layershell::Error> {
+    Panel::run(Settings {
```

Comment:
I mean Panel::run already return `Result<(), iced_layershell::Error>`

look at this

https://github.com/waycrate/exwlshelleventloop/blob/master/iced_examples/counter/src/main.rs#L16-L25

Reply:
Yes yes got it, my bad